.github/workflows/dependency-review.yml

```
@@ -10,7 +10,7 @@
 name: 'Dependency review'
 on:
   pull_request:
-    branches: [ "master", "next" ]
+    branches: [ "master", "next", "wip" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)